contracts/VotiumBribesProcessor.sol

```diff
@@ -210,12 +214,15 @@ contract VotiumBribesProcessor is CowSwapSeller {
 
             toEmit = totalBveCVX - ops_fee;
 
+            // Take fee
             IERC20(address(BVE_CVX)).safeTransfer(TREASURY, ops_fee);
-            IERC20(address(BVE_CVX)).safeTransfer(BADGER_TREE, toEmit);
         }
 
+        // Emit token
+        IERC20(address(BVE_CVX)).safeApprove(address(HARVEST_FORWARDER), toEmit);
+        HARVEST_FORWARDER.distribute(address(BVE_CVX), toEmit, address(BVE_CVX));
```

Comment:
A good sanity check before distribution is to assert `HARVEST_FORWARDER.badger_tree == BADGER_TREE`